articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -24,6 +24,8 @@ This article guides you through how to create an Azure [dedicated host](dedicate
 ## Limitations
 
 - The sizes and hardware types available for dedicated hosts vary by region. Refer to the host [pricing page](https://aka.ms/ADHPricing) to learn more.
+- Not all Azure VM SKUs, regions and availability zones support Ultra disks, see [Azure ultra disks](disks-enable-ultra-ssd.md) for more information.
+- The fault domain count of the virtual machine scale set can't exceed the fault domain count of the host group.
```

Comment:
This change is already live on the docs, would it be best to sync your branch with the main repo so we don't have conflicts?

Reply:
yeah, forgot to pull into my fork from the main branch!